Sources/Camille/Karma/KarmaService+Patterns.swift

```diff
@@ -24,7 +24,7 @@ extension KarmaService {
             case .topUsers: return ["top", Int.any.using(key: Keys.count)]
             case .myCount: return ["how much karma do I have"]
             case .userCount: return ["how much karma does", User.any.using(key: Keys.user), "have"]
-            case .adjustment: return [User.any, ["++", "--"].any]
+            case .adjustment: return [User.any, ["++", "+=1", "+= 1", "--", "-=1", "-= 1"].any]
```

Comment:
I think it would be good for maintenance if we moved this array onto the `Operations` enum as a computed property.. something like:
```
var values: [String] { return ... }
```

Might be a good idea to also remove `private` from the enum and maybe nest it inside `KarmaService`, so:
```
extension KarmaService {
   enum Operations {
      ..
   }
}
```
then the pattern would become return [User.any, Operations.values.any]
```